app/assets/javascripts/application.js

```diff
@@ -375,16 +355,9 @@ function spinner_placeholder(text){
   return "<div class='spinner-placeholder'><p class='spinner-label'>" + text + "</p><div id='Loading' class='spinner spinner-md spinner-inline'> </div></div>";
 }
 
-function notify(item, type) {
-  var icon = typeToIcon(type);
-  var options = { classMessage: "foreman-alert",
-                  classBackground: "",
-                  sticky: (type != 'success'),
-                  type: type };
-  $.jnotify("</div>" + icon + $(item).text(), options);
-  // jnotify does not support multiple classes passed via classMessage so added via jquery.
-  $('.foreman-alert').addClass("alert alert-" + type);
-  $(item).remove();
+function notify(message, type) {
+  tfm.tools.deprecate('notify', 'tfm.toastNotifications.notify');
+  tfm.toastNotifications.notify({message: message, type: type});
```

Comment:
here we pass the parameters as a hash, but the function expects regular parameters.